clouds/snowflake/modules/sql/h3/H3_KRING.sql

```diff
@@ -1,30 +1,15 @@
 ----------------------------
--- Copyright (C) 2021 CARTO
+-- Copyright (C) 2021-2024 CARTO
 ----------------------------
 
-CREATE OR REPLACE FUNCTION @@SF_SCHEMA@@._H3_KRING
-(origin STRING, size DOUBLE)
-RETURNS ARRAY
-LANGUAGE JAVASCRIPT
-IMMUTABLE
-AS $$
-    if (SIZE == null || SIZE < 0) {
-        throw new Error('Invalid input size')
-    }
-
-    @@SF_LIBRARY_H3_KRING@@
-
-    if (!h3KringLib.h3IsValid(ORIGIN)) {
-        throw new Error('Invalid input origin')
-    }
-
-    return h3KringLib.kRing(ORIGIN, parseInt(SIZE));
-$$;
-
 CREATE OR REPLACE SECURE FUNCTION @@SF_SCHEMA@@.H3_KRING
-(origin STRING, size INT)
+(h3_hex STRING, distance INT)
```

Comment:
For the distance occurs the same that for `index` previously. Don't know if I would keep size or distance.
What do you think @ernesmb?

In any case if we decide to go for distance this should be updated in other provider too and also in every doc to be consistent.

Reply:
For context: My main reason for changing these is that 'index' and 'size' are reserved keywords in SQL. 

As for the specific new terms I chose: 
The 'distance' word is what is used in the official Uber H3 libraries. Example:  https://uber.github.io/h3-py/api_reference#h3.great_circle_distance

And I use either h3_hex or h3_int in any function which takes a H3 index to differentiate between the hex and int representation of H3 cells.

Reply:
To me at least changing size -> distance sounds oki.

Regarding `index`-> `h3_hex`/`h3_int` not that sure.

cc: @Jesus89